src/components/Meta.tsx

```diff
@@ -34,6 +35,10 @@ export function Meta({ title, description, imageUrl, timestamp }: MetaProps) {
         key="keywords"
         content="Decentralized Finance, DeFi, lending, borrowing, stablecoins, Ethereum, assets, erc-20, smart contracts, open finance, trustless"
       />
+      <link rel="apple-touch-icon" href={imageUrl180} />
```

Comment:
You could just hardcode the image path here, then no need to add an optional prop. Should be the same everywhere this component is used.